update-arm-images.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+[[ -z "${CLIENT_ID}" ]] && echo "CLIENT_ID not set" >&2 && exit 1;
+[[ -z "${CLIENT_SECRET}" ]] && echo "CLIENT_SECRET not set" >&2 && exit 1;
+
+# Pull base images once to avoid pull limit in dockerhub
+docker pull ubuntu:18.04
+
+while IFS= read -r VERSION
+do
+if ! [[ "$VERSION" < "7.0.0" ]]; then
```

Comment:
Consider doing `>=` instead of `! <` . Could be lexicographical issues though, so check

Reply:
That would be `if [[ "$VERSION" > "7.0.0" || "$VERSION" = "7.0.0" ]]; then` which is a bit more verbose, not sure what is easier to read. maybe keep the first with just a comment to make it more clear?